app/views/dresses/show.html.erb

```diff
@@ -21,17 +21,17 @@
     <p>Size: <%= @dress.size %></p>
     <p>Color: <%= @dress.color %></p>
     <p>Price Per Day: $<%= @dress.price_per_day %>
+    <%= link_to 'Book Dress', new_dress_booking_path(@dress), class: 'btn btn-dark  float-right' %>
```

Comment:
Having a simple form with the starting date and ending date can be used here instead. I think this might not pass validation otherwise    

<%= simple_form_for [ @dress, @booking ] do |f| %>
      <%= f.input :starting_date, as: :string, required: false, input_html: {id: "start_date"} %>
      <%= f.input :ending_date, as: :string, required: false, input_html: {id: "end_date"} %>
      <%= f.submit class: 'btn btn-dark float-right'%>
    <% end %>

Reply:
![Uploading Screenshot 2022-01-27 at 00.16.51.png…]()

Reply:
![Screenshot 2022-01-27 at 00 16 51](https://user-images.githubusercontent.com/84076465/151191028-d60d6d29-189f-4990-9da7-df02e8f3ad68.png)

Reply:
This simple form version looks good

Reply:
I already tested it out though, and it works fine? I mean, this is just a link to the simple form for new booking.

Reply:
This is just a link to the booking form that you made :) which is already a simple form with the same fields

Reply:
<img width="393" alt="Screen Shot 2022-01-27 at 9 25 48" src="https://user-images.githubusercontent.com/86721807/151269190-baadfdd6-9013-4de6-bd0f-65807038a609.png">

after you click the button it takes you here


<img width="1377" alt="Screen Shot 2022-01-27 at 9 25 54" src="https://user-images.githubusercontent.com/86721807/151269196-882417b1-8d88-4cf7-b50d-c28e27b36529.png">

Reply:
Okay, but are we getting rid of the new page and just putting it on the dressing show page as a form?

Reply:
I feel like a button looks cleaner? It is a bit unusual to have the whole checkout process displayed on an item show page perhaps?

Reply:
That's cool and I'll merge it now ^_^